src/models/validators/TestResultsSchemaHGVCancelled.ts

```diff
@@ -78,4 +78,10 @@ export const hgvCancelled = testResultsCommonSchema.keys({
     .required(),
   testTypes: Joi.array().items(testTypesSchema).required(),
   regnDate: Joi.string().allow('', null),
+  recalls: Joi.object()
+    .keys({
+      hasRecall: Joi.boolean().required(),
+      manufacturer: Joi.string().required().allow('', null),
```

Comment:
When would the value of manufacturer ever be an empty string? I would expect it, if it's not present to be sent as a null

Reply:
Discussed in PR chat last week and agreed to allow null and empty string.